lib/hmmlearn/tests/test_base.py

```diff
@@ -60,20 +59,36 @@ def _compute_log_likelihood(self, X):
 class TestBaseAgainstWikipedia:
     def setup_method(self, method):
         # Example from http://en.wikipedia.org/wiki/Forward-backward_algorithm
-        self.framelogprob = np.log([[0.9, 0.2],
-                                    [0.9, 0.2],
-                                    [0.1, 0.8],
-                                    [0.9, 0.2],
-                                    [0.9, 0.2]])
+        self.frameprob = np.asarray([[0.9, 0.2],
+                                     [0.9, 0.2],
+                                     [0.1, 0.8],
+                                     [0.9, 0.2],
+                                     [0.9, 0.2]])
+        self.framelogprob = np.log(self.frameprob)
 
         h = StubHMM(2)
-        h.transmat_ = [[0.7, 0.3], [0.3, 0.7]]
-        h.startprob_ = [0.5, 0.5]
+        h.transmat_ = np.asarray([[0.7, 0.3], [0.3, 0.7]])
```

Comment:
Does it not work anymore to set these as lists?  If so perhaps the implementation should be adjusted to support such inputs.

Reply:
Good catch.  I have undid this change and wrapped calls into Cython where needed with `np.asarray`